### ISSUE TITLE
JS : Can not add cookie

### ISSUE THREAD

**Issue by dominiquefournier** [craigmiskell/cookiemaster#13]
Hi 
Since FF68.4.1 Linux Debian, I am not able to set/unset a cookie with Javascript when the extension is activated. When I disable it, it works well, even in strict mode.
Steps to reproduce : Use the code : 
```
<script>
console.log ("Before", document.cookie);
document.cookie = "AdminCookie=on;expires=Sat, 08 Feb 2020 09:33:57 GMT;path=/"
console.log ("In", document.cookie);
document.cookie = "AdminCookie=;expires=Thu, 21 Aug 2014 20:00:00 UTC;path=/"
console.log ("After", document.cookie);
</script>
```
In console, without extension : 
```
Before
In AdminCookie=on
After
```
in console, with extension, even if the site is allowed in list : 
```
Before
In
After 
```
Do you have an idea how to set the cookie ?
Thanks

**Comment by craigmiskell** [comment]
Ugh, another bug with the new javascript cookie code as of v1.1.0 of cookiemaster.  I'll see what I can do to fix it.

**Comment by craigmiskell** [comment]
Just released 1.1.3 which should deal with this
